FLAnimatedImageDemo/FLAnimatedImage/FLAnimatedImage.m

```diff
@@ -288,17 +284,19 @@ - (instancetype)initWithAnimatedGIFData:(NSData *)data
                         FLLogInfo(@"Rounding frame %zu's `delayTime` from %f up to default %f (minimum supported: %f).", i, [delayTime floatValue], kDelayTimeIntervalDefault, kDelayTimeIntervalMinimum);
                         delayTime = @(kDelayTimeIntervalDefault);
                     }
-                    delayTimesMutable[i] = delayTime;
+                    delayTimesForIndexesMutable[@(i)] = delayTime;
                 } else {
+                    skippedFrameCount++;
                     FLLogInfo(@"Dropping frame %zu because valid `CGImageRef` %@ did result in `nil`-`UIImage`.", i, frameImageRef);
                 }
                 CFRelease(frameImageRef);
             } else {
+                skippedFrameCount++;
                 FLLogInfo(@"Dropping frame %zu because failed to `CGImageSourceCreateImageAtIndex` with image source %@", i, _imageSource);
             }
         }
-        _delayTimes = [delayTimesMutable copy];
-        _frameCount = [_delayTimes count];
+        _delayTimesForIndexes = [delayTimesForIndexesMutable copy];
+        _frameCount = imageCount;
```

Comment:
By setting `_frameCount` to imageCount, we're includes skipped frames. Is that what you want? Or should it reflect the total count minus the skipped frames? I see before the change we were counting the total count as well, so this is likely fine.